src/openrct2-ui/interface/Window.cpp

```diff
@@ -435,9 +435,9 @@ static void window_viewport_wheel_input(rct_window* w, int32_t wheel)
 {
     if (gScreenFlags & (SCREEN_FLAGS_TRACK_MANAGER | SCREEN_FLAGS_TITLE_DEMO))
         return;
-    if (wheel < 0 && !gConfigGeneral.disable_scroll_zoom)
+    if (wheel < 0 && !gConfigGeneral.scroll_wheel_behaviour == 0)
```

Comment:
Did you test this? `!gConfigGeneral.scroll_wheel_behaviour == 0` will bind the `!` operator to `gConfigGeneral.scroll_wheel_behaviour`, inverting the setting rather than the expression.

I'd preferred for the entire block to be nested in one `gConfigGeneral.scroll_wheel_behaviour != 0` if-block.